js/ui/components/trayArea.js

```diff
@@ -118,15 +118,15 @@ var TrayArea = new Lang.Class({
 
         // Saturation
         if (arguments.length == 1) {
-            let sat_effect = new Clutter.DesaturateEffect({ factor : ICON_SATURATION });
-            sat_effect.set_factor(ICON_SATURATION);
-            sat_effect.set_factor(ICON_SATURATION);
+            let sat_effect = new Clutter.DesaturateEffect({ factor : ICON_DESATURATION });
```

Comment:
For consistency, should probably call this variable "desat_effect"

Reply:
I agree